package.json

```diff
@@ -74,7 +74,7 @@
     "pre-commit": "^1.2.2",
     "prettier": "^1.11.1",
     "standard-version": "^4.3.0",
-    "webpack": "^4.28.0",
+    "webpack": "4.28.0",
```

Comment:
Why?

Reply:
Seemingly upgraded to webpack 4.29 and caused all tests with the import() syntax to fail. Locking it at least makes stuff pass. Not specifically a webpack problem, but incredibly irritating. Can try tinkering with it again

Reply:
Finally got the green check ✅ with this though. I’ll check on the status of the npm-cli release. Hoisting supposed to be patched in its next release.